salon_creator/lib/screens/login_page.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:salon_creator/authentication/sign_in.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key key}) : super(key: key);
+
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(children: [
+        new Container(
+          decoration: new BoxDecoration(
+            image: new DecorationImage(
+              image: new AssetImage("assets/background.jpg"),
+              fit: BoxFit.fill,
+            ),
+          ),
+        ),
```

Comment:
Maybe a good idea to refactor L17-24 into a separate method **_buildBackgroundImage** and L25-48 into **_buildLoginButtons**